jupiterone/modifiers.go

```diff
@@ -180,6 +181,65 @@ func (j jsonIgnoreDiff) PlanModifyList(ctx context.Context, req planmodifier.Lis
 	resp.PlanValue = req.StateValue
 }
 
+// PlanModifyMap implements planmodifier.Map
+func (jsonIgnoreDiff) PlanModifyMap(ctx context.Context, req planmodifier.MapRequest, resp *planmodifier.MapResponse) {
+	if req.ConfigValue.IsNull() {
+		return
+	}
+
+	if req.StateValue.IsUnknown() || req.StateValue.IsNull() {
+		return
+	}
+
+	oldValues := req.StateValue.Elements()
+	for k, v := range req.PlanValue.Elements() {
+		o, ok := oldValues[k]
+		if !ok {
+			// something new, go ahead with the plan
+			return
+		}
+
+		s, ok := v.(types.String)
+		if !ok {
+			// this is very bad and shouldn't haven't gotten past validation
```

Comment:
Is there any kind of error we should return or publish?

Reply:
> Is there any kind of error we should return or publish?

I have added a warning for this to the diagnostics.